.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Thank you for submitting a PR to the rust rpm implementation!
+-->
+
+# PR Content Desc
+
+<!---
+DELETE all that do not apply:
+-->
+
+- 🩹 Bug Fix
+- 🦚 Feature
+- 📙 Documentation
+- 🦣 Legacy
+- 🪣 Misc
+
+<!---
+Mention the linked issue here.
+This will automatically close the issue once the PR is merged
+and creates a cross reference.
+-->
+
+Closes #
+
+## Changes proposed by this PR
+
+<!---
+Tell the reviewer WHAT changed, and WHY it had to change
+and how the WHAT and the WHY tie together.
+-->
+
+## Notes to reviewer
+
+<!---
+Leave a message to whoever is going to review this PR.
+Mainly, pointers to review the PR, and how they can test it.
+If things are still WIP or feedback on particular impl details
+are wanted, state them here too.
+-->
+
+## 📜 Checklist
+
+- [ ] Test coverage is excellent and passes
+- [ ] Works when tests are run `--all-features` enabled
```

Comment:
tests are run *and* `--all-features` enabled?

As it stands the `test-with-podman` test makes this difficult to properly test. I'd rather have this be a "CI passes" check.

Reply:
The CI part does not work, due to containerization, which last time I checked, does not work.

Reply:
```suggestion
- [ ] Works when tests are run with `--all-features` enabled
```